### PR TITLE
Modify the order in which verify-maas.yml is run

### PR DESCRIPTION
**Pull request by cloader89**
This moves the running of verify-maas.yml to the last item run so
that it ensures that everything is set up before checking.

Connects #1448

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/staging.horse/build-be3a3b86d0/docs/private-cloud/release-notes/). :bow:

File: scripts/deploy-rpc-playbooks.sh [hunk 1]
```diff
@@ -29,3 +28,9 @@ if [[ "${DEPLOY_ELK}" == "yes" ]]; then
     run_ansible haproxy.yml
   fi
 fi
+
+#verify RAXA MaaS is running after all necessary
```
Comment (sigmavirus24) [inline scripts/deploy-rpc-playbooks.sh]: Nit: All comments should have spaces after the octothorpe. Also what is "RAXA" ;)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/staging.horse/build-c3846e6592/docs/private-cloud/release-notes/)

File: scripts/deploy-rpc-playbooks.sh (hunk 1 above)
Comment (cloader89) [inline scripts/deploy-rpc-playbooks.sh]: fixed

**Comment by d34dh0r53** [comment]
:+1: looks good to me, thanks for fixing the nits.

**Comment by cloader89** [comment]
20:57:02 cp: cannot stat ‘/etc/rackspace-monitoring-agent.cfg’: No such file or directory

recheck_swift

**Pull request merged**